apps/design/backend/src/worker/worker.ts

```diff
@@ -0,0 +1,46 @@
+import { extractErrorMessage, sleep } from '@votingworks/basics';
+
+import { BackgroundTask, Store } from '../store';
+
+async function processBackgroundTask(
+  _store: Store, // eslint-disable-line @typescript-eslint/no-unused-vars
+  _task: BackgroundTask // eslint-disable-line @typescript-eslint/no-unused-vars
+): Promise<void> {
+  // Simulate a long-running operation
+  // TODO(arsalan): Implement actual processing logic
+  await sleep(3000);
+}
+
+async function processBackgroundTasks(store: Store): Promise<void> {
+  // eslint-disable-next-line no-constant-condition
+  while (true) {
+    const queuedTasks = store.getQueuedBackgroundTasks();
```

Comment:
Nit: Since we only ever want one task at a time, maybe we just query for a single task here?

Reply:
👍 I like that: https://github.com/votingworks/vxsuite/pull/4162/commits/89d6761bdf1f0ecefaabf9024db90512887facf3